spec/index.html

```diff
@@ -1152,6 +1152,23 @@ <h3>The XML Schema Built-in Datatypes</h3>
         model.</li>
     </ul>
 
+    <p class="note">The <a>value spaces</a>
+      of <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a> and
+      <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a> do not
+      include all decimal numbers. The reason is that, for every literal of
```

Comment:
As suggested by @afs in https://github.com/w3c/rdf-concepts/pull/44#discussion_r1201874881
```suggestion
      include all decimal numbers. For every literal of
```